nstars.c

```diff
@@ -157,7 +157,7 @@ int main(int argc, char *argv[])
 					beep();
 				}
 				break;
-			case 10:					/* Enter */
+			case KEY_ENTER:
```

Comment:
[Yes][cint]. I do, I can never remember correctly which ASCII value belongs to which.

[cint]: http://stackoverflow.com/a/18422933/242583

[st][st] is "simple terminal" of suckless.

[st]: http://st.suckless.org/